include/z64animation.h

```diff
@@ -25,7 +25,7 @@ typedef enum {
     /* 3 */ ANIMMODE_ONCE_INTERP,
     /* 4 */ ANIMMODE_LOOP_PARTIAL,
     /* 5 */ ANIMMODE_LOOP_PARTIAL_INTERP
-} AnimationModes;
+} AnimationModes; // AnimationInfo mode
```

Comment:
Is there a particular reason this comment was added?

Reply:
Got tired of searching for that enum and not finding it, but yeah its not a proper tag and I'll remove

Reply:
You can add a comment above the enum about what it's used for, or better is probably to add a documentation comment above the structs that usually use it.